### DISCUSSION TITLE
How to update on Raspberry Pi ?

### DISCUSSION THREAD

**Discussion by Kriss1670** [teslamate-org/teslamate#2325] [Q&A]
I'm sorry but even if not noob, I can't find how to update my TeslaMate installed on a Raspberry Pi 4...
If anyone can help ?

By the way would be great to have auto-updating from the user interface.

**Comment by cwanja** [comment]
https://docs.teslamate.org/docs/upgrading

**Comment by Kriss1670** [comment]
Hello,

Thanks for answering but I had already seen this page and it doesn’t work on my installation… « git » is an unknown command.
I inform you that I’m not a noob on computers. I developped some softwares and use Raspberry Pi for other usages including PiHole, Home Assistant, Minecraft Server… It really would be a great feature that the system upgrades almost automatically.

If you have another solution, I would appreciate.
Thanks.

**Reply by cwanja** [reply]
If `git` is an unknown command, sounds like you need to install git on your Pi. 

Probably worth sharing how you installed TeslaMate as well to further help you. To be fair, at least for PiHole and Home Assistant, I do not think either allow for UI upgrade processes. All are required through the CLI.

**Comment by Kriss1670** [comment]
Home Assistant updates automatically… but this is not a bad comment on your work, just a suggestion ! 😉
I can’t remember how I installed TeslaMate. That was almost 2 years ago and at this time, I was just beginning with Raspberry. Will try to find out with the help of a friend.

Thank you.

De : cwanja
Envoyé le :mardi 18 janvier 2022 15:54
À : adriankumpf/teslamate
Cc : Kriss1670; Author
Objet :Re: [adriankumpf/teslamate] How to update on Raspberry Pi ?(Discussion #2325)

If git is an unknown command, sounds like you need to install git on your Pi.
Probably worth sharing how you installed TeslaMate as well to further help you. To be fair, at least for PiHole and Home Assistant, I do not think either allow for UI upgrade processes. All are required through the CLI.
—
Reply to this email directly, view it on GitHub, or unsubscribe.
Triage notifications on the go with GitHub Mobile for iOS or Android.
